### PR TITLE
testing readme examples

### PR DESCRIPTION
**Pull request by Borda**
# Before submitting

- [x] Was this discussed/approved via a Github issue? (no need for typos and docs improvements)
- [x] Did you read the [contributor guideline](https://github.com/PyTorchLightning/pytorch-lightning/blob/master/.github/CONTRIBUTING.md), Pull Request section?
- [x] Did you make sure to update the docs?   
- [x] Did you write any new necessary tests?  

## What does this PR do?

add testing readme examples, would be very welcome to get this feat https://github.com/tmarktaylor/phmdoctest/issues/5

## PR review    
Anyone in the community is free to review the PR once the tests have passed.     
If we didn't discuss your PR in Github issues there's a high chance it will not be merged.

## Did you have fun?
Make sure you had fun coding 🙃

### CODE REVIEW

**Comment by Borda** [comment]
waiting for https://github.com/tmarktaylor/phmdoctest/pull/6 be merged and released :]

File: .github/workflows/ci_test-full.yml [hunk 1]
```diff
@@ -90,6 +79,9 @@ jobs:
 
     - name: Tests
       run: |
+        phmdoctest README.md --outfile tests/test_readme.py
+        # fixme: remove this print
+        cat tests/test_readme.py
```
Comment (Borda) [inline .github/workflows/ci_test-full.yml:84]:
```suggestion
```

**Review by tchaton — APPROVED** [review]
Like this ! We can do this for all repos.

**Comment by Borda** [comment]
> Like this ! We can do this for all repos.

yes, this is PoC, TM is a kind of entry gate for other PL projects...

**Review by justusschock — APPROVED** [review]

**Pull request merged**